Index: minipress.api/src/models/Article.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\ndeclare(strict_types=1);\r\n\r\nnamespace minipress\\api\\models;\r\n\r\nuse Illuminate\\Database\\Eloquent\\Model as Eloquent;\r\n\r\n\r\nclass Article extends Eloquent\r\n{\r\n        protected $table = \"article\";\r\n    protected $primaryKey = \"id_article\";\r\n    public $timestamps = false;\r\n    public $incrementing = false;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/minipress.api/src/models/Article.php b/minipress.api/src/models/Article.php
--- a/minipress.api/src/models/Article.php	(revision 094a682c524c73265299f83f7bb223ed9edbf3ab)
+++ b/minipress.api/src/models/Article.php	(date 1686831467550)
@@ -9,7 +9,7 @@
 
 class Article extends Eloquent
 {
-        protected $table = "article";
+    protected $table = "article";
     protected $primaryKey = "id_article";
     public $timestamps = false;
     public $incrementing = false;
